### PR TITLE
OUPDAUNTLE-56 add support for custom alfresco mounts

### PR DESCRIPTION
**Pull request by JoostDeCupere**
Also updated the chart to support 3fold image references

### CODE REVIEW

**Review by thijslemmens — CHANGES_REQUESTED** [review]
After discussion:

- mq-deployement.yaml -> mq-deployment.yaml
- make 3-fold image the default and have 'image' override that

File: README.md [hunk 1]
```diff
@@ -279,11 +279,29 @@ For more information take a look at
...
+
+* Required: false
+* Default: None
+* Description: The tag of the docker image that will be used
+
 #### `acs.image`
 
-* Required: true
+* Required: true if no acs.registry, acs.repository or acs.tag is provided
 * Default: xenit/alfresco-repository-community:7.2.0
```
Comment (thijslemmens) [inline README.md:303]: default is unset?

**Review by thijslemmens — CHANGES_REQUESTED** [review]
The `image` documentation needs to be changed. There are no defaults anymore.

**Review by thijslemmens — APPROVED** [review]
@JoostDeCupere Can you check my updates before merging?

File: xenit-alfresco/values.yaml [hunk 2]
```diff
@@ -26,7 +26,9 @@ ingress:
 
 acs:
   replicas: 1
-  image: 'xenit/alfresco-repository-community:7.2.0'
+  registry: 'xenit'
```
Comment (hechmi-dammak-xenit) [inline xenit-alfresco/values.yaml:29]: I think this should be docker.io and the repository is xenit/alfresco-repository-community , same for the other ones

File: xenit-alfresco/values.yaml [hunk 3]
```diff
@@ -53,7 +55,9 @@ acs:
 digitalWorkspace:
   enabled: true
   replicas: 1
-  image: 'quay.io/alfresco/alfresco-digital-workspace:2.4.2-adw'
+  registry: 'quay.io/alfresco'
```
Comment (hechmi-dammak-xenit) [inline xenit-alfresco/values.yaml:58]: and this is quay.io and the repository is alfresco/alfresco-digital-workspace

File: xenit-alfresco/values.yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ general:
   db: {}
   cni: cilium
   networkPolicies:
-    enabled: true
+    enabled: false
```
Comment (hechmi-dammak-xenit) [inline xenit-alfresco/values.yaml:10]: why is this disabled

**Review by hechmi-dammak-xenit — CHANGES_REQUESTED** [review]

**Review by hechmi-dammak-xenit — APPROVED** [review]

**Pull request merged**